test/persistence-hooks.suite.js

```diff
@@ -86,6 +86,18 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           });
       });
 
+      it('should trigger access hook once, if near filter is used', function(done) {
+        monitorHookExecution();
+        TestModel.find(
+          { where:
+            { location: { near: '10,20', maxDistance: '10', unit: 'meters' }},
+          }, function(err, list) {
+          if (err) return done(err);
+          hookMonitor.names.should.eql(['access']);
```

Comment:
Please also note I do not think we have ever had `loaded` hook if `near` filter was used. We may add it in another PR.